back/CinemaReservation.BusinessLayer/Contracts/ISecurityService.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace CinemaReservation.BusinessLayer.Contracts
+{
+    public interface ISecurityService
+    {
+        byte[] GetHash(string password, byte[] salt);
+        (byte[] passwordHash, byte[] salt) GetPasswordHashAndSalt(string password);
+        bool IsCorrectPassword(byte[] passwordHash, byte[] salt, string passwordToCheck);
```

Comment:
Only self-properties can start with Is according to convention. Methods should start with verb. Expected name CheckPasswordCorrectness